src/app/component/shared/detail-view/view/view.component.ts

```diff
@@ -3,29 +3,44 @@ import { DateTime, Duration } from "luxon";
 import { toRelative } from "src/app/interfaces/apiInterfaces";
 
 @Component({
-  selector: "app-view",
+  selector: "app-render-view",
```

Comment:
```suggestion
  selector: "baw-render-field",
```

We are not doing anything with a "view" - we're just displaying the value of a field